src/domain/users/phone-metadata-validator.ts

```diff
@@ -4,11 +4,21 @@ import {
   InvalidCarrierForPhoneMetadataError,
   InvalidCarrierTypeForPhoneMetadataError,
   InvalidCountryCodeForPhoneMetadataError,
+  InvalidErrorCodeForPhoneMetadataError,
+  InvalidMobileCountryCodeForPhoneMetadataError,
 } from "./errors"
 
+const checkedToCarrierType = (rawCarrierType: string | undefined | null): CarrierType => {
+  if (rawCarrierType == null || rawCarrierType == undefined) {
+    return "" as CarrierType
+  }
+
+  return rawCarrierType as CarrierType
```

Comment:
this should validate Object.values(CarrierType).includes(type)

Reply:
https://github.com/GaloyMoney/galoy/pull/3276/commits/ccbd455fd3cb7ca5a475cf5f959db6b5087bb189